app/scripts/constants/stream.ts

```diff
@@ -0,0 +1,16 @@
+// contexts
+export const CONTENT_SCRIPT = 'metamask-contentscript';
+export const METAMASK_INPAGE = 'metamask-inpage';
+export const PHISHING_WARNING_PAGE = 'metamask-phishing-warning-page';
+
+// stream channels
+export const METAMASK_PROVIDER = 'metamask-provider';
+export const METAMASK_COOKIE_HANDLER = 'metamask-cookie-handler';
```

Comment:
```suggestion
export const METAMASK_COOKIE_HANDLER = 'metamask-cookie-handler';
export const METAMASK_PROVIDER = 'metamask-provider';
```
just a nit: swap the 2 so it's ordered alphabetically